dockerfiles/Dockerfile_full

```diff
@@ -9,7 +9,7 @@ RUN mkdir /usr/local/lib/nodejs \
   && tar xf nodejs.tar.gz -C /usr/local/lib/nodejs/ --strip-components 1 \
   && rm nodejs.tar.gz node-file-lock.sha
 RUN apt-get install --assume-yes \
-  openjdk-11-jdk-headless=11.0.7+10-2ubuntu2~18.04 \
+  openjdk-11-jdk-headless \
```

Comment:
I think we will get the latest version but simply not pointing to a specific release in the script.

https://www.oracle.com/java/technologies/javase-downloads.html

https://www.digitalocean.com/community/tutorials/how-to-install-java-with-apt-on-ubuntu-18-04